src/plugins/data_source_management/public/components/data_source_menu/data_source_selectable.tsx

```diff
@@ -16,14 +16,15 @@ import {
 } from '@elastic/eui';
 import { SavedObjectsClientContract, ToastsStart } from 'opensearch-dashboards/public';
 import { getDataSourcesWithFields } from '../utils';
-import { DataSourceOption, LocalCluster } from '../data_source_selector/data_source_selector';
+import { LocalCluster } from '../data_source_selector/data_source_selector';
 import { SavedObject } from '../../../../../core/public';
 import { DataSourceAttributes } from '../../types';
+import { DataSourceOption } from './types';
 
 interface DataSourceSelectableProps {
   savedObjectsClient: SavedObjectsClientContract;
   notifications: ToastsStart;
-  onSelectedDataSource: (dataSource: DataSourceOption) => void;
+  onSelectedDataSources: (dataSources: DataSourceOption[]) => void;
```

Comment:
any context of the renaming from `onSelectedDataSource` to `onSelectedDataSources` ? MDS drop down doesn't support multiple selected right ? (aka we can only select one data source at a time)

Reply:
We are supporting multi select which is in another PR, thus changing it to selected data sources